locales/en/plugin__odf-console.json

```
@@ -1096,6 +1097,14 @@
   "NamespaceStore details": "NamespaceStore details",
   "Target Blob Container": "Target Blob Container",
   "Num Volumes": "Num Volumes",
+  "Create Bucket": "Create Bucket",
+  "An object bucket is a cloud storage container that organizes and manages files (objects), allowing users to store, retrieve and control access to data efficiently.": "An object bucket is a cloud storage container that organizes and manages files (objects), allowing users to store, retrieve and control access to data efficiently.",
+  "Select bucket creation method": "Select bucket creation method",
+  "Create via S3 API": "Create via S3 API",
+  "Ideal for applications and systems that need to interact directly with S3 storage, especially when integrating with AWS-native services and features.": "Ideal for applications and systems that need to interact directly with S3 storage, especially when integrating with AWS-native services and features.",
```

Didn't notice it in the mockups, but this has nothing to do with "AWS-native". Added a comment on designs as well.
```suggestion
  "Ideal for applications and systems that need to interact directly with S3 storage, especially when integrating with AWS-native services and features.": "Ideal for applications and systems that need to interact directly with S3 storage.",
```

---

Until further discussion, we'll go with this text: `We'll go with this text: 'Ideal for applications and systems that need to interact directly with S3-compatible  storage.'`